modules/oe_webtools_maps/src/Plugin/Field/FieldFormatter/WebtoolsMapFormatter.php

```diff
@@ -64,10 +65,17 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
 
     foreach ($items as $delta => $item) {
       $element[$delta] = [
-        '#theme' => 'oe_webtools_maps_map',
-        '#latitude' => $item->get('lat')->getValue(),
-        '#longitude' => $item->get('lon')->getValue(),
-        '#zoom_level' => $this->getSetting('zoom_level'),
+        '#type' => 'html_tag',
+        '#tag' => 'script',
+        '#value' => new JsonEncoded([
```

Comment:
Just curious, why you don't use this:
```suggestion
        '#value' => Json::encode([
```
I suspect that this decision based on security reasons, but could you at least comment this?